crates/compiler/can/src/expr.rs

```diff
@@ -348,6 +349,12 @@ pub enum Expr {
     RuntimeError(RuntimeError),
 }
 
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub enum TryKind {
```

Comment:
This will eventually be removed once we remove the `try` keyword, so consider it temporary